source/_posts/to-1.0-and-beyond.md

```diff
@@ -0,0 +1,104 @@
+
+But hey, we are a company with a much ambitious vision!
+It is time for us to build a solid community of teams sharing our love for a better, faster web.
+
+That’s why we are happy to officially introduce Karim, our new team member!
+Karim is in charge of Business Development &amp; Partnership since more than a month now.
+
+> Hi everyone! I’m Karim, a ………………………………….
+> Nice to meet you! Let’s get in touch……………
+
+Karim and Pierre will attend [OroVibe France](https://orovibe.com/2019france/en) next week, it would be a good way to say us hi!
+
+## `1.0.0-alpha.0`, `1.0.0-alpha.1`… every release brings us closest to `1.0.0`!
+
+Until 2018, we have been _dogfooding_ and incubating Front-Commerce by delivering several projects in production as part of Occitech.
+
+Armed with experience from these real-world use-cases, we started internal refactorings and reorganization to allow Front-Commerce to be used with confidence by different teams over time.
+These « big bang » backward incompatible changes have been our first priority to ensure that the projects which were being started would not need tedious rewrites when upgrading.
+
+Since our first `alpha` release we have been working on incremental improvements, reducing technical debt, and will continue until we are confident that we could follow [our roadmap](/docs/appendices/roadmap.html) with semantic versioning in mind.
+
+We invite you to read our [releases page](https://gitlab.com/front-commerce/front-commerce/releases) (if you have access to our private Gitlab instance), or our [Migration Guides](https://developers.front-commerce.com/docs/appendices/migration-guides.html).
+We will announce new releases on this blog too from now on.
+
+## Dig into our documentation
+
+This is definitely one of the most long-awaited resource.
+Partners, potential customers and developers have been asking it for a while and it was one of the first thing we wanted to deliver to them/you.
+It marks our confidence in our current API and architecture (we did not want to document things that would radically change), and the start for a continuous improvement.
+
+Our code is closed source, but our culture is open source! This is why we wanted to make our official documentation public (some API references may still be restricted though).
+
+**We are proud to announce our revamped version of https://developers.front-commerce.com** (this site), [almost 1 year](/blog/2018/03/15/hello-world/) after publishing its first version.
+As every documentation, this is an ongoing process.
+We are commited to spend 1 or 2 days a week to publish new pages, and have « documented » in our [Definition Of Done](https://www.agilealliance.org/glossary/definition-of-done/) so every new feature or change will come with new content in our documentation.
```

Comment:
This is not clear that the `1 or 2 days` is separated from the Definition Of Done. Adding a link work that emphasizes the separation between the two parts of the sentence might do the trick.